package.json

```diff
@@ -5,11 +5,12 @@
   "main": "dist/node/pill.js",
   "devDependencies": {
     "babel-minify": "^0.5.0",
+    "copy": "^0.3.2",
     "rollup": "^1.7.4",
     "serve": "^10.1.2"
   },
   "scripts": {
-    "start": "serve -l 8080 -n example",
+    "start": "npm run build && node copy-files.js && serve -l 8080 -n example",
```

Comment:
It should use `build:iife` to avoid redundant, in case of example build, commonjs building and minification.